roles/openafs_client/tasks/redhat/install-build.yaml

```diff
@@ -84,10 +84,24 @@
     path: /tmp/openafs-client-build/usr/afs
     state: absent
 
-- name: Install files
-  command: rsync -av * /
-  args:
-    chdir: /tmp/openafs-client-build
+# Needed to split the install into separate tasks
+# in order to get correct result,
+# /lib/modules was not updating correctly
```

Comment:
I think i'd prefer to have this commentary in the commit message.

Reply:
Ah, i see that commentary is in the commit message already. I'll merge as is, and we can clean this up later.